@AH-64D Apache Longbow/Addons/fza_ah64_fcr/functions/fn_stateControl.sqf

```diff
@@ -22,56 +22,61 @@ Author:
 #include "\fza_ah64_systems\headers\systems.hpp"
 params ["_heli"];
 
-private _fcrDamage     = _heli getHitPointDamage "hit_msnequip_fcr";
-private _lastScanState = _heli getVariable "fza_ah64_fcrLastScan";
-private _acBusOn       = _heli getVariable "fza_systems_acBusOn";
-private _dcBusOn       = _heli getVariable "fza_systems_dcBusOn";
-private _fcrState      = _heli getVariable "fza_ah64_fcrState";
-Private _fcrMode       = _heli Getvariable "fza_ah64_fcrMode";
-private _armaRadarOn   = isVehicleRadarOn _heli;
-private _updateDelay   = [4,7.7] select (_fcrMode == 2);
-private _onGnd         = [_heli] call fza_sfmplus_fnc_onGround;
+private _fcrDamage   = _heli getHitPointDamage "hit_msnequip_fcr";
+private _acBusOn     = _heli getVariable "fza_systems_acBusOn";
+private _dcBusOn     = _heli getVariable "fza_systems_dcBusOn";
+Private _fcrMode     = _heli Getvariable "fza_ah64_fcrMode";
```

Comment:
`private` and `getVariable` - You need to check around as this is wrong in a few places AFAIK

Reply:
I must have had cap locks on - corrected